ConsoleCommands.cs

```diff
@@ -452,8 +452,7 @@ public void OnRestartMatchCommand(CCSPlayerController? player, CommandInfo? comm
                 }
                 else
                 {
-                    // ReplyToUserCommand(player, "Practice mode is active, cannot restart the match.");
-                    ReplyToUserCommand(player, Localizer["matchzy.cc.rrispracc"]);
+                    Server.ExecuteCommand("mp_restartgame 1")
```

Comment:
You forgot a `;` 

`                    Server.ExecuteCommand("mp_restartgame 1");`

Reply:
added